packages/mwp-core/src/util/inlineStyle.test.js

```diff
@@ -0,0 +1,22 @@
+import { getInlineStyleTags } from './inlineStyle.js';
+
+// `simple-universal-style-loader` reades from `global`,
+// so we must mock the data directly in `global.__styles__`
+global.__styles__ = [
```

Comment:
might be nicer to do this before the `it` and then clear it with `global.__styles__ = null` after the test so you don't leak